README.md

```diff
@@ -37,21 +34,93 @@ $ ./spin --help
 
 After you follow the [quickstart document](https://developer.fermyon.com/spin/quickstart/),
 you can follow the
-[Rust](https://developer.fermyon.com/spin/rust-components/) or [Go](https://developer.fermyon.com/spin/go-components/)
+[Rust](https://developer.fermyon.com/spin/rust-components/), [JavaScript](https://developer.fermyon.com/spin/javascript-components), [Python](https://developer.fermyon.com/spin/python-components), or [Go](https://developer.fermyon.com/spin/go-components/)
 language guides, and the [guide on configuring Spin applications](https://developer.fermyon.com/spin/configuration/).
 
-After you build your application, run it using Spin:
+Below is an example of using the `spin` CLI to create a new Spin Python application, then adding a JavaScript component:
+
+```bash
+# Create a new Spin application based on the Python language template.
+$ spin new http-py hello-python
+# Add a new JavaScript component based on the language template.
+$ spin add http-js goodbye-javascript
+```
+
+Running the `spin add` command will generate the proper configuration for our component and add it to the [`spin.toml` manifest file](https://developer.fermyon.com/spin/manifest-reference). For example, here is the `spin.toml` section for our Python component:
 
+```toml
+[[component]]
+# The ID of the component.
+id = "hello-python"
+# The Wasm module to instantiate and execute when receiving a request.
+source = "hello-python/app.wasm"
+[component.trigger]
+# The route for this component.
+route = "/hello"
+[component.build]
+# The command to execute for this component with `spin build`.
+command = "spin py2wasm app -o app.wasm"
+# The working directory for the component.
+workdir = "hello-python"
 ```
+
+We can now build our application with `spin build`, then run it locally with `spin up`:
```

Comment:
This skips the plugin installation steps, so anyone using this as a quick start is going to get weird errors... again my suggestion would be to link to the docs rather than trying to write a separate quickstart here.  Could we just hit the highlights here, in a way that is visibly "here is the 30000 ft view" rather than looking like a tutorial but actually being incomplete?

Reply:
I revamped the example to make it simpler. It is now just a quick spin new, spin build, spin up.